src/pages/authentication/Login.jsx

```diff
@@ -0,0 +1,137 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import { useNavigate } from "react-router";
+import { Formik } from "formik";
+import * as Yup from "yup";
+
+import { Input } from "../../customComponents/Form/Input";
+import { loginFormChangeHandler, loginFormSubmitHandler } from "../../utils";
+
+import "./Authentication.css";
+import { useAuth } from "../../hooks";
+import { useNotifications } from "reapop";
+
+const Login = () => {
+  const navigate = useNavigate();
+  const { notify } = useNotifications();
+
+  const [formData, setFormData] = useState({
+    email: "",
+    password: "",
+  });
+  const { authState, authDispatch } = useAuth();
+
+  const validator = Yup.object({
+    email: Yup.string()
+      .matches(/^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/, "Enter valid email")
+      .required("Required"),
+    password: Yup.string()
+      .min(6, "Must be 6 to 15 characters")
+      .max(15, "Must be 6 to 15 characters")
+      .required("Required"),
+  });
```

Comment:
Like the validator thing. Seeing this yup package for the first time. Will definitely explore more about this one. Tnanks.